google/cloud/debugger/quickstart/quickstart.cc

```diff
@@ -17,16 +17,14 @@
 #include <stdexcept>
 
 int main(int argc, char* argv[]) try {
-  if (argc != 3) {
-    std::cerr << "Usage: " << argv[0] << " debuggee-id client-version\n";
+  if (argc != 2) {
+    std::cerr << "Usage: " << argv[0] << " project-id\n";
     return 1;
   }
 
-  auto const debuggee_id = argv[1];
-  auto const client_version = argv[2];
   namespace debugger = ::google::cloud::debugger;
   auto client = debugger::Debugger2Client(debugger::MakeDebugger2Connection());
-  auto response = client.ListBreakpoints(debuggee_id, client_version);
+  auto response = client.ListDebuggees(argv[1], "quickstart");
```

Comment:
Should this be a `"project-id"`, or a `Project("project-id").FullName()`?

Reply:
I was surprised too, I tested with both, and it should be a "project-id" **without** the `projects/` prefix

Reply:
_"Project number of a Google Cloud project whose debuggees to list."_  Yikes.  If it really is a "project-number", perhaps we should change the usage message?

Reply:
Lucky for me, it is not a project number.  The field name in the request is simply `project` and of type string:

https://github.com/googleapis/googleapis/blob/9f7048a87f13215ff066f1bdf6e3e95c933414e5/google/devtools/clouddebugger/v2/debugger.proto#L203

And as I said, I tested (manually) with a project id, it works, it returns a non-empty list:


```
bazel run //google/cloud/debugger/quickstart:quickstart -- cloud-cpp-testing-resources
INFO: Analyzed target //google/cloud/debugger/quickstart:quickstart (0 packages loaded, 0 targets configured).
INFO: Found 1 target...
Target //google/cloud/debugger/quickstart:quickstart up-to-date:
  /usr/local/google/home/coryan/.cache/bazel/_bazel_coryan/f1b649d8eddab70da217bfe95503f270/execroot/com_github_googleapis_google_cloud_cpp/bazel-out/k8-fastbuild/bin/google/cloud/debugger/quickstart/quickstart
INFO: Elapsed time: 4.595s, Critical Path: 4.34s
INFO: 3 processes: 1 internal, 2 linux-sandbox.
INFO: Build completed successfully, 3 total actions
INFO: Build completed successfully, 3 total actions
debuggees {
  id: "gcp:936212892354:214fa12f45549f4a"
  project: "936212892354"
  uniquifier: "da39a3ee5e6b4b0d3255bfef95601890afd80709"
  description: "cloud-cpp-testing-resources-20220215t192634"
```

I am planning to deactivate the app, at which point it will continue to work but return an empty list (I tested that too).

I am happy with this change as-is.

Reply:
> Lucky for me, it is not a project number. The field name in the request is simply `project` and of type string:
> 
> https://github.com/googleapis/googleapis/blob/9f7048a87f13215ff066f1bdf6e3e95c933414e5/google/devtools/clouddebugger/v2/debugger.proto#L203

A string no doubt, but "Project number" seems clear about what that string should contain.

> I am happy with this change as-is.

Sounds good to me.  Just a comment.